articles/libraries/machine_learning/concepts/Qccc.Introduction.md

```diff
@@ -0,0 +1,33 @@
+
+# Classifier architecture
+Classification is a supervised machine learning task, where the goal is to infer class labels $\{y_1,y_2,\ldots,y_d\}$ of certain data samples. The `training data set` is a collection of samples $\mathcal{D}=\{(x,y)}$ with known pre-assigned labels. Here $x$ is a data sample and $y$ is its known label called `training label`.
+Somewhat similar to traditional methods, quantum classification consists of three steps: data encoding, preparing of a classifier state, taking a measurement. Due to probabilistic nature of the measurement, these three steps must be repeated multiple times. The measurement may be viewed as a quantum equivalent of some non-linear activation.
+Both the encoding and the computing of the classifier state are done by means of `quantum circuits`. While the encoding circuit is usually data-driven and parameter-free, the classifier circuit contains a sufficient set of learnable parameters. In the proposed solution the classifier circuit is composed of single-qubit rotations and two-qubit controlled rotations. The learnable parameters here are the rotation angles. The rotation and controlled rotation gates are known to be `universal` for quantum computation, which means that any unitary weight matrix can be decomposed into a long enough circuit consisting of such gates. Our architecture explores relatively shallow circuits, which therefore must be `rapidly entangling` in order to capture all the correlations between the data features at all ranges. An example of the most useful rapidly entangling circuit component is shown on FIG 1. Even though this circuit consists of only $3\,n+1$ gates, the unitary weight matrix that it computes ensures significant cross-talk between $2^n$ features.
+ 
+
+## FIG. 1
+![Rapidly entangling quantum circuit on 5 qubits (with two cyclic layers).](./5qubitCircuit.png)
+
+The circuit consists of 6 single-qubit gates $(G_1,\ldots,G_5; G_{16})$ and 10 two-qubits gates $(G_6,\ldots,G_{15})$. Assuming that each of the gates is defined with one learnable parameter we have 16 learnable parameters, while the dimension of the 5-qubit Hilbert space is 32. Such circuit geometry can be easily generalized to any $n$-qubit register, when $n$ is odd, yielding circuits with $3\, n+1$ parameters for $2^n$-dimensional feature space.
+
+# Classifier training as a supervised learning task
+Training of a classifier model involves finding optimal values of its operational parameters, such that they maximize the average likelihood of inferring the correct training labels across the training samples.
+Here  we concern ourselves with two level classification only, i.e. the case of $d=2$ and only two classes with the labels $y_1,y_2$. \footnote{A principled way of generalizing our methods to arbitrary number of classes is to replace qubits with q-dits, i.e. quantum units with $d$ basis states, end the two-way measurement with $d$-way measurement.}
```

Comment:
LaTeX notation is only used by our DocFX configuration for formatting math, such that `\footnote` isn't recognized. A sidebar may be a more natural way to format this note.